### ISSUE TITLE
add RowGroup example to documentation site?

### ISSUE THREAD

**Issue by edavidaja** [rstudio/DT#505]
I'm not sure what I'm doing wrong, but I can't get the row groups to display, so it would be helpful to have an example to work from.

Here's what I've tried, with both the latest CRAN and github versions, to no avail:
```
datatable(mtcars, extensions = "RowGroup", options = list(dataSrc = 'cyl'))
datatable(mtcars, extensions = "RowGroup", options = list(dataSrc = 2))
```
refs #442

**Comment by SmarakNayak** [comment]
Looking at the DataTables website you can see [here](https://datatables.net/extensions/rowgroup/) that `dataSrc` needs to be within a `rowGroup` object. So the following should work:

`datatable(mtcars, extensions = "RowGroup", options = list(rowGroup=list(dataSrc=2)))`

**Comment by philibe** [comment]
An working example inspired from the last comment and from the js of  [datatables.net :  customRow](https://datatables.net/extensions/rowgroup/examples/initialisation/customRow.html)  :

```
output$my_output <- DT::renderDataTable(
    DT::datatable(my_output_datas(),
                   style = "bootstrap",   class = "compact", filter='top',
                   extensions = "RowGroup",
                   selection = c("single"),    
                   options = list(
                     bSortClasses = TRUE, iDisplayLength = 10,   width = "100%",
                     lengthMenu  = list(c(5, 25, 50, 75, 100, -1), list('5', '25','50','75','100', 'All')),
                     list(order = list(6, 'asc')),
                     rowGroup=list(
                        startRender= NULL,
                        endRender= JS ("function ( rows, group ) {
                                         var ageAvg =  rows.count();
                                         return $('<tr/>')
                                         .append( '<td colspan=\\'13\\'>Averages for '+group+'</td>' )
                                         .append( '<td>'+ageAvg.toFixed(0)+'</td>' );
                                       } "
                       ),
                       dataSrc=6
                       )
                   )
    )   
  )
````

**Comment by shrektan** [comment]
Closed by #757
